src/components/Modal/index.js

```diff
@@ -1,69 +1,98 @@
-import VanillaModal from 'vanilla-modal'
 import { events } from 'onfido-sdk-core'
 import style from './style.css'
+import ReactModal from 'react-modal'
+import { h, render, Component } from 'preact'
+import { getCSSMilisecsValue, wrapWithClass, impurify } from '../utils'
 
-const Modal = {}
+const MODAL_ANIMATION_DURATION = getCSSMilisecsValue(style.modal_animation_duration)
```

Comment:
`react-modal` needs a timeout for close if one wants to apply a closing animation, hence the variable sharing with css.
The reason for the timeout is because react-modal actually unmounts the dom elements when the modal is closed, it's not just a simple css hiding behaviour.